share/server/dreyfus.js

```diff
@@ -45,10 +45,10 @@ var Dreyfus = (function() {
     indexDoc: function(doc) {
       Couch.recursivelySeal(doc);
       var buf = [];
-      for each (fun in State.funs) {
+      for (fun in State.funs) {
```

Comment:
Should we define `fun` here? It looks like it may be getting assigned to a global. `var fun` would do the trick.

Reply:
Good call. Let me change it to `fun_idx`